File: src/Passenger.java

public class Passenger extends Person {
    private double bagID;
    private double bagWeight;

    public Passenger(String name, String surname, int idNumber, Phone phone, Date birthday, String gender, double weight, double height, double bagID, double bagWeight) {
        super(name, surname, idNumber, phone, birthday, gender, weight, height);
        this.bagID = bagID;
        this.bagWeight = bagWeight;
    }

    public double getBagID() {
        return bagID;
    }
    public void setBagID(double bagID) {
        this.bagID = bagID;
    }
    public double getBagWeight() {
        return bagWeight;
    }
    public void setBagWeight(double bagWeight) {
        this.bagWeight = bagWeight;
    }
    @Override
    public String toString() {
        return "Passenger [bagID=" + bagID + ", bagWeight=" + bagWeight + "]";
    }



}

File: webclient/src/app/models/passenger.ts
import { Flight } from './flight';

export class Passenger
{
     id:number;
     name: string;
     surname: string;
     phone: number;
     birthday:Date;
     gender: string;
     weight: number;
     height: number;
     flightHistory: Flight[];
     bagId: number;
     bagWeight:number;


    public constructor(id:number,name:string,surname:string,phone:number,birthday:Date,gender:string,weight:number,
                        height:number,bagId:number,bagWeight:number)
    {
        this.id = id;
        this.name = name;
        this.surname = surname;
        this.phone = phone;
        this.birthday = birthday;
        this.gender = gender;
        this.weight = weight;
        this.height = height;
        this.bagId = bagId;
        this.bagWeight = bagWeight;
    }

}